File: 1-Beginner/1017/bee1017.c
// 1017 - Gasto de Combustível
#include <stdio.h>

int main(void) {

    //12km/L
    int horas, vel;
    scanf("%d%d", &horas, &vel);

    int distancia = vel*horas;

    double litros = (double)distancia/12;

    printf("%.3f\n", litros);

    return 0;
}

File: 1-Beginner/1017/bee1017.java
// 1017 - Gasto de Combustível
import java.io.IOException;
import java.util.Scanner;

class bee1017 {

    public static void main(String[] args) throws IOException {
        Scanner scan = new Scanner(System.in);

        //12km/L
        int horas, vel;
        horas = scan.nextInt(); vel = scan.nextInt();

        int distancia = vel*horas;

        double litros = (double)distancia/12;

        System.out.printf("%.3f\n", litros);

        scan.close();
    }
}